src/budy/controllers/api/order.py

```
@@ -247,6 +281,104 @@ def ctt_csv(self):
+    @appier.route("/api/orders/inventory.pdf", "GET")
+    @appier.ensure(token="admin")
+    def inventory_pdf(self):
+        start = self.field("start", cast=int)
+        end = self.field("end", cast=int)
+        paid = self.field("paid", True, cast=bool)
+        thumbnail = self.field("thumbnail", True, cast=bool)
+        view = self.field("view")
+        context = self.field("context")
+        headless_url = appier.conf("BUDY_HEADLESS_URL", "https://headless.bemisc.com")
+        headless_key = appier.conf("BUDY_HEADLESS_KEY", None)
+        appier.verify(not headless_key == None)
```

Ruff (0.15.11)

[error] Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)